File: backend/app/Models/PriceList.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;
use App\Traits\HasLog;

class PriceList extends Model
{
    use SoftDeletes, HasLog;

    protected $fillable = ['name', 'account_id'];

    public function items()
    {
        return $this->hasMany(PriceListItem::class);
    }

    public function account()
    {
        return $this->belongsTo(Account::class);
    }
}

File: compose-app/shared/src/commonMain/kotlin/com/sales/app/domain/model/PriceList.kt
package com.sales.app.domain.model

import kotlinx.serialization.Serializable

@Serializable
data class PriceList(
    val id: Long,
    val name: String,
    val itemsCount: Int = 0,
    val items: List<PriceListItem> = emptyList()
)
